tools/mo/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -1995,7 +1995,7 @@ def test_mo_convert_params(self):
         assert argv.transform == "LowLatency2[use_const_initializer=False]"
 
         for arg, value in vars(argv).items():
-            if arg not in args:
+            if arg not in args and arg != 'is_python_api_used':
```

Comment:
why do we need this change? Is check not missed?

Reply:
In this cycle we check that pack_params_to_args_namespace() sets default values for all params, except of values from "args", because they should have non-default values and are checked above. And "is_python_api_used" is skipped, because it is a helper value and does not have a default value.